.github/workflows/dart.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         - uses: actions/checkout@v2
         - run: pub get
         - name: Test on Chrome
-          run: pub run test -p chrome test/*
+          run: pub run test -p chrome .
```

Comment:
when we run on `test/*` Dart try to run test on each file in the folder, but test_utils.dart is not a test file.